Libraries/Components/TextInput/TextInputNativeCommands.js

```diff
@@ -22,6 +22,14 @@ export interface TextInputNativeCommands<T> {
     start: Int32,
     end: Int32,
   ) => void;
+
+  // [macOS
+  // NYI on Android
+  +setGhostText: (
+    viewRef: React.ElementRef<T>,
+    value: ?string, // in theory this is nullable
+  ) => void;
+  // macOS]
 }
 
 const supportedCommands = ['focus', 'blur', 'setTextAndSelection'];
```

Comment:
You don't need to add `setGhostText` here?

Reply:
no one uses it? But okay, added in case we use it in the future.